website/schema2Markdown.js

```diff
@@ -350,11 +350,15 @@ function retrieveTitleFromSchemaData(schemaData, pathInSchema) {
 }
 
 function parseSchemaFolder(schemaFolderName) {
+    const schemaFolder = `./static/schemas/2.1/${schemaFolderName}`;
```

Comment:
hardcoding 2.1 into the generated schema paths. Needs to be 'next' instead. The versioned schemas are locked and don't change. Those in 'next' can change with a PR then get locked down when we run the versioning script when preparing a final release.

Reply:
There's a function looking to scan for the folder name "context". That folder only exists within the folder 2.1. I can't change this to "next", because then the rest of the function breaks, because the files related to context don't exist in the "next" folder.

Should the "next" folder be updated? Should we handle the fact that some of the schemas folder don't have *.schema.json files? Please advise.

Reply:
The folder doesn't exist in the /website/static/schemas/next folder as this script was broken:
https://github.com/InteropIO/FDC3/blob/241a0f7902eab80b1465ddefb5b38df710200d2a/website/package.json#L9

The second path needs to change, probably to:

```
"copy-schemas": "del-cli static/schemas/next/*/ && cpy \"../packages/fdc3-schema/schemas**\" static/schemas/next/ && cpy \"../packages/fdc3-context/schemas**\" static/schemas/next/",
```

(schemas folder got split into two different packages)

The `copy-appd` script probably needs to change to:

```
 "copy-appd": "cpy \"../packages/fdc3-standard/src/app-directory/specification/*.schema.json\" \"static/schemas/next/\"",
```

Reply:
I tested and fixed this